webui/frontend/src/app/summarize-history/page.tsx

```diff
@@ -1,14 +1,91 @@
-import React from 'react';
-import Header from '../components/Header';
+"use client";
+import React, { useState } from "react";
+import Header from "../components/Header";
+import { useQuery } from "react-query";
+import HistoryCard from "../components/HistoryCard"
+import { data } from "./summarizeHistoryMock";
+import SummarizeHistoryModal from "./SummarizeHistoryModal"
+import { SummaryTask } from "../types/TaskType";
+
 
 const SummarizeHistory = () => {
+
+  const [selectedTaskIndex, setSelectedTaskIndex] = useState<number | null>(null);
+
+  const handleCardClick = (index: number) => {
+    setSelectedTaskIndex(index);
+  };
+
+  const closeModal = () => {
+    setSelectedTaskIndex(null);
+  };
+  console.log(selectedTaskIndex);
+  console.log(data.task[0])
```

Comment:
Are these logs necessary? It's not clear what they're outputting with no label attached.